src/App.js

```diff
@@ -12,15 +12,19 @@ class App extends React.PureComponent {
       <Router>
         <div>
           <nav>
-            <h1>BookStore CMS</h1>
+            <h1 className="logo">BookStore CMS</h1>
             <ul className="links">
               <li>
                 <Link to="/">Books</Link>
               </li>
               <li>
                 <Link to="/categories">Categories</Link>
               </li>
+              <div>
+                <div className="oval"><img src={`${process.env.PUBLIC_URL}images/profile-icon.svg`} alt="profile-icon" /></div>
+              </div>
             </ul>
+
           </nav>
```

Comment:
- [ ] According to the requirements, Styling should be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) 💯 so kindly use the same colors for the nav links and remove the underline 👍

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/204219946-a41308bd-b2cc-4804-96c5-8e4c76fb7da8.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/204219936-42373d86-8880-4361-b6ac-7b5f7a7b0268.PNG" width="500" /> |